app/src/main/res/layout/continue_playing_fragment.xml

```diff
@@ -2,18 +2,26 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <data>
+
+    <import type="org.oppia.app.R" />
+
+    <variable
+      name="presenter"
+      type="org.oppia.app.home.HomeFragmentPresenter" />
```

Comment:
I don't think we want to pull in HomeFragmentPresenter here, plus this variable doesn't seem to be used. Please remove.

Reply:
Removed.